feature-libs/cart/saved-cart/styles/_add-to-saved-cart.scss

```diff
@@ -1,9 +1,29 @@
 cx-add-to-saved-cart {
+  max-height: 70px;
+
   .cx-add-to-saved-cart-container {
     display: flex;
-    justify-content: center;
+    justify-content: start;
     margin-top: 3rem;
 
+    .cx-action-link {
+      font-size: 1rem;
```

Comment:
These values are overriding the Design System's `.cx-action-link` style. All action links across Spartacus should look the same, as per the modified screenshot:

![Screen Shot 2021-03-15 at 3 22 33 PM](https://user-images.githubusercontent.com/15113219/111209825-e4843d80-85a2-11eb-8cd2-87df49b841f0.png)

Reply:
> Approving **styles** but Keyboard Accessibility needs to be reviewed and popover repositioning doesn't seem to work as per screenshots, please coordinate with the topic owner for this feature.
> ![Screen Shot 2021-03-15 at 5 42 22 PM](https://user-images.githubusercontent.com/15113219/111226578-b65d2880-85b7-11eb-97a1-7a72eb162fbe.png)
> ![Screen Shot 2021-03-15 at 5 48 52 PM](https://user-images.githubusercontent.com/15113219/111226580-b6f5bf00-85b7-11eb-977f-e2d6292decc1.png)

not part of this and it's a known issue. The popover is related to other team's feature which Michal and the team on it are looking to fix